theme/templates/accounts/profile.html

```diff
@@ -4,78 +4,516 @@
+                                    <td>{{ profile.subject_areas }}</td>
+                                </tr>
+                            {% endif %}
 
-            {% if profile.organization %}
-            <li><strong>Organization:</strong> {{ profile.organization }}</li>
-            {% endif %}
+                            {% if profile.profession %}
+                                <tr>
+                                    <td><span class="text-muted">Profession</span></td>
+                                    <td>{{ profile.profession }}</td>
+                                </tr>
+                            {% endif %}
+                        {% endif %}
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
 
-            {% if profile.organization_type %}
-            <li><strong>Organization Type:</strong> {{ profile.organization_type }}</li>
-            {% endif %}
+        <div class="full-width-tabs-container">
+            <!-- Nav tabs -->
+            <ul class="nav nav-tabs full-width-tabs" role="tablist">
+                <li role="presentation" class="active">
+                    <a href="#overview" aria-controls="home" role="tab"
+                       data-toggle="tab"><i class="glyphicon glyphicon-list-alt"></i> OVERVIEW</a></li>
+                <li role="presentation">
+                    <a href="#contributions" aria-controls="profile" role="tab" data-toggle="tab">
+                        <i class="glyphicon glyphicon-file"></i> CONTRIBUTIONS</a>
+                </li>
+{#                <li role="presentation">#}
+{#                    <a href="#social" aria-controls="messages" role="tab"#}
+{#                       data-toggle="tab"><i class="glyphicon glyphicon-user"></i> SOCIAL</a></li>#}
+            </ul>
 
-            {% if profile.phone_1 %}
-            <li><strong>Phone 1:</strong> {{ profile.phone_1 }} {{ profile.phone_1_type }}</li>
-            {% endif %}
+            <!-- Tab panes -->
+            <div class="tab-content">
+                <div role="tabpanel" class="tab-pane fade in active" id="overview">
+                    <div class="row">
+                        <div class="col-md-7">
+                            <div class="panel panel-default">
+                                <div class="panel-heading">
+                                    <h3 class="panel-title">Recent activity</h3>
+                                </div>
+                                <div class="panel-body">
+                                    {% if resources %}
 
-            {% if profile.phone_2 %}
-            <li><strong>Phone 2:</strong> {{ profile.phone_2 }} {{ profile.phone_2_type }}</li>
-            {% endif %}
+                                    {% for res in resources reversed %}
+                                        {# Show only the last five #}
+                                        {% if forloop.counter0 < 5 %}
```

Comment:
@pkdash could you take a look and make sure the resource list is sorted by time? If not, could you help @mauriel133 to provide such backend support? It might be better to provide another resource subset context variable just for this purpose.